components/dashboard/sections/expenses/ReceivedExpenses.tsx

```diff
@@ -147,6 +147,7 @@ const ReceivedExpenses = ({ accountSlug }: DashboardSectionProps) => {
       account: { slug: accountSlug },
       fetchHostForExpenses: false, // Already fetched at the root level
       hasAmountInCreatedByAccountCurrency: false,
+      hasGrantHistory: false,
```

Comment:
🍨 The `fetchGrantHistory` pattern seems more explicit with the function of this variable.